packages/pipcook-core/src/index.ts

```diff
@@ -9,4 +9,5 @@ export {PipcookRunner} from './core/core';
 export {createAnnotationFile, parseAnnotation, unZipData, downloadZip, getOsInfo, transformCsv,
   getDatasetDir, createAnnotationFromJson, getMetadata, getModelDir, convertPascol2CocoFileOutput, compressTarFile } from './utils/publicUtils';
 export {startBoard} from './board/board';
-export {PipcookComponentResult} from './types/component';
\ No newline at end of file
+export {PipcookComponentResult} from './types/component';
+export {PLUGINS} from './constants/plugins';
```

Comment:
We should export `constants` only at the top-level just like `constants.PLUGINS` :)

Reply:
Yes I did, please take a look:)